### DISCUSSION TITLE
Is it possiblie to trigger `useElementVisibility` once?

### DISCUSSION THREAD

**Discussion by abdul-alhasany** [vueuse/vueuse#4007] [General]
It seems that `useElementVisibility` triggers every time the element appears in viewport regardless of how many times the user scrolls up or down the page.
I think it is uesfull to add an option to trigger once. This is a common behaviour when animate elements on scroll. 
I have looked through the documentation and code but I don't think this option exists.

Is there a way to do it? or is there another function that does this?

**Comment by ilyaliao** [comment]
Maybe we can improve this, feel free to open an issue, happy to resolve it :)

-----------

PR: #4577

**Comment by abdul-alhasany** [comment]
Thanks for the PR. I had everything ready for the PR except the test, I could not figure it out.
